src/glass/src/app/pages/users-page/users-page.component.ts

```diff
@@ -0,0 +1,210 @@
+            {
+              type: 'text',
+              label: TEXT('Name'),
+              name: 'username',
+              value: '',
+              validators: {
+                required: true
+              }
+            },
+            {
+              type: 'text',
+              label: TEXT('Full Name'),
+              name: 'full_name',
+              value: ''
+            },
+            {
+              type: 'password',
+              label: TEXT('Password'),
+              name: 'password',
+              value: '',
+              validators: {
+                required: true
+              }
+            },
+            {
+              type: 'checkbox',
+              label: TEXT('Disabled'),
+              name: 'disabled',
+              value: false,
+              hint: TEXT('Temporarily prohibit the user from logging in.')
+            }
+          ],
+          submitButtonText: TEXT('Add')
+        } as DeclarativeFormModalConfig
+      }
+    );
+  }
+
+  onActionMenu(user: User): DatatableActionItem[] {
+    const result: DatatableActionItem[] = [
+      {
+        title: TEXT('Edit'),
+        callback: (data: DatatableData) => {
+          this.dialogService.open(
+            DeclarativeFormModalComponent,
+            (res: User | boolean) => {
+              if (res) {
+                this.blockUI.start(translate(TEXT('Please wait, updating user ...')));
+                this.userService
+                  .update((res as User).username, res as User)
+                  .pipe(finalize(() => this.blockUI.stop()))
+                  .subscribe(() => {
+                    this.loadData();
+                  });
+              }
+            },
+            {
+              width: '40%',
+              data: {
+                title: TEXT('Edit User'),
+                fields: [
+                  {
+                    type: 'text',
+                    label: TEXT('Name'),
+                    name: 'username',
+                    value: user.username,
+                    readonly: true
+                  },
+                  {
+                    type: 'text',
+                    label: TEXT('Full Name'),
+                    name: 'full_name',
+                    value: user.full_name
+                  },
+                  {
+                    type: 'password',
+                    label: TEXT('Password'),
+                    name: 'password',
+                    value: ''
+                  },
+                  {
+                    type: 'checkbox',
+                    label: TEXT('Disabled'),
+                    name: 'disabled',
+                    value: user.disabled,
+                    hint: TEXT('Temporarily prohibit the user from logging in.')
+                  }
+                ],
+                submitButtonText: TEXT('Edit')
+              } as DeclarativeFormModalConfig
+            }
+          );
+        }
+      },
+      {
+        title: TEXT('Delete'),
```

Comment:
Can also be addressed in a separate issue: I do have the feeling we should prevent the logged-in user from deleting himself.

Reply:
The API endpoint will check that.

Reply:
:+1: I think it would be great if the user **can't** disable himself in addition.